contracts/interchain-token-service/src/contract/execute.rs

```diff
@@ -153,11 +157,8 @@ pub fn get_token_id(deps: &DepsMut, denom: &str) -> Result<TokenId, Error> {
         .change_context(Error::FailedTokenIdGeneration)?;
     let chain_name_hash: [u8; 32] = Keccak256::digest(chain_name.to_string().as_bytes()).into();
 
-    let gateway_token_prefix: [u8; 32] =
-        Keccak256::digest("its-interchain-token-id-gateway").into();
-
     let token_id_raw: [u8; 32] =
-        Keccak256::digest([&gateway_token_prefix, &chain_name_hash, denom.as_bytes()].concat())
+        Keccak256::digest([&gateway_token_prefix(), &chain_name_hash, denom.as_bytes()].concat())
             .into();
```

Comment:
if you wanna avoid defining a variable, although not sure if it's really better
```suggestion
            .then([u8; 32]::from)
            .into()
            .then(Ok);
```